implementations/python/sugar-lift-py-tests/tests/test_exit_set_normalize_index.py

```
@@ -0,0 +1,362 @@
+    import pkgutil
+    import typing
+
+    import sugar_lift_py_tests as package
+    from sugar_lift_py_tests.effect import Effect
+    from sugar_lift_py_tests.floor import FloorValue
+
+    for module in pkgutil.walk_packages(package.__path__, package.__name__ + "."):
+        try:
+            importlib.import_module(module.name)
+        except BaseException:  # optional/heavy leaves are not this tooth's job
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)